src/doc/src/reference/registry-index.md

```diff
@@ -139,10 +139,13 @@ explaining the format of the entry.
             // https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html.
             "req": "^0.6",
             // Array of features (as strings) enabled for this dependency.
+            // May be omitted since Cargo 1.85.
             "features": ["i128_support"],
-            // Boolean of whether or not this is an optional dependency. Defaults to `false` if not specified.
+            // Boolean of whether or not this is an optional dependency.
+            // Since Cargo 1.85, defaults to `false` if not specified.
             "optional": false,
-            // Boolean of whether or not default features are enabled. Defaults to `true` if not specified.
+            // Boolean of whether or not default features are enabled.
+            // Since Cargo 1.85, defaults to `true` if not specified.
```

Comment:
Thanks!

cargo@0.85.0 is equivalent to 1.84.

The 1.85 beta hasn't yet branched off (here is [the schedule](https://forge.rust-lang.org/#current-release-versions)), and I'll do a Cargo submodule update in rust-lang/rust later today, so I assume it is still 1.84.